src/mame/seta/seta.cpp

```diff
@@ -11359,6 +11386,7 @@ GAME( 1993, msgundamb, msgundam, msgundamb, msgundam,  seta_state,     empty_ini
 
 GAME( 1993, oisipuzl,  0,        oisipuzl,  oisipuzl,  seta_state,     empty_init,     ROT0,   "Sunsoft / Atlus",           "Oishii Puzzle Ha Irimasenka", 0 )
 GAME( 1993, triplfun,  oisipuzl, triplfun,  oisipuzl,  seta_state,     empty_init,     ROT0,   "bootleg",                   "Triple Fun", 0 )
+GAME( 1993, triplfunk, oisipuzl, triplfun,  oisipuzl,  seta_state,     empty_init,     ROT0,   "bootleg (Jin Young)",       "Sumeoinneun Deongdarireul Chajara!", 0 ) // 숨어있는 덩달이를 찾아라! (Find the hiding Deongdari!)
```

Comment:
Same here, I don’t think the title is correct.

Reply:
Can you please copy the exact transliteration and Korean to use? I've asked again to the owner, Korean native, and he keep saying that what we used is right.

Thanks!!